compiler/options.nim

```diff
@@ -118,6 +118,12 @@ var
   gNoNimblePath* = false
   gExperimentalMode*: bool
 
+# enable colors by default on terminals
+from terminal import isatty
+
+if terminal.isatty(stdout):
+  incl(gGlobalOptions, optUseColors)
+
```

Comment:
I like this idea but don't do this in options.nim please. Instead do it in msgs.nim.